README.md

```diff
@@ -9,33 +9,40 @@
 
 ## Milestone 1 README
 
-For this project, we will be creating R packages that implement k-means
-clustering from scratch. This will work on any dataset with valid
-numerical features and includes fit, predict, and summarize functions,
-as well as elbow and silhouette methods for hyperparameter “k”
-optimization.
+This package will include R packages that implement k-means clustering
+from scratch. This will work on any dataset with valid numerical
+features, and includes fit, predict, and score functions, as well as as
+elbow and silhouette methods for hyperparameter “k” optimization.
 
-  - fit: Outputs a list of cluster centers based on the inputted dataset
-    and number of clusters. Only clustered based on valid numerical
-    features.
+  - fit: This function classifies the non-labeled data into a given
+    number of clusters k using simple KMeans algorithm. It returns
+    labels for each data point according to the cluster it belongs and
+    also cluster centers. This is a type of unsupervised learning method
+    to classify data.
 
   - predict: Assigns each point in a dataset to a cluster. Dataset has
     to be in the same format as the original dataset the model was fit
     on.
 
-  - cluster\_summary: Outputs a summary of clusters created including
-    centroid coordinates, number of data points in training data
-    assigned to each cluster, and within-cluster distance metrics.
+  - score: Outputs a score based on goodness of fit.
```

Comment:
please add some measure of the goodness of fit we are using. like silhouette or elbow.